tests/hotplug/affinity.go

```diff
@@ -32,6 +37,9 @@ var _ = Describe("[sig-compute]VM Affinity", decorators.SigCompute, decorators.S
 	)
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
+		kv := util2.GetCurrentKv(virtClient)
+		kv.Spec.Configuration.VMRolloutStrategy = (*v1.VMRolloutStrategy)(pointer.P(v1.VMRolloutStrategyLiveUpdate))
```

Comment:
this cast can be dropped by defining  `VMRolloutStrategyLiveUpdate` as `VMRolloutStrategy`.
See above.